src/exchanges/gemini-client.js

```diff
@@ -59,12 +68,14 @@ class GeminiClient extends EventEmitter {
     if (!subscription) {
       subscription = {
         market,
-        wss: this._connect(remote_id),
+        wss: this._connect(remote_id, mode),
```

Comment:
since tickers uses a special `&top_of_book` url query param I had to pass in the mode as an argument so it can know when to add that extra param